styles.css

```diff
@@ -310,9 +309,105 @@ nav {
 }
 
 .lang > li > h3 {
-  font-family: 'Poppins', sans-serif;
+  font-family: "Poppins", sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 15px;
   line-height: 20px;
 }
+
+.container {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.section {
+  background-image: url(images/form-img.png);
+  background-position: center right;
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  height: 779px;
+  width: 375px;
```

Comment:
- [OPTIONAL] It is not recommended to use fixed widths and heights or `vh` and `vw` to dimension sections like this because the user can open this portfolio through different mobile phones and different screen sizes and this does not make the page responsive. Your portfolio in an iPhone XR looks like this:
 -
<img width="476" alt="image" src="https://user-images.githubusercontent.com/61919617/192565267-a47c28d2-f9b2-4175-bf9b-9b865da49bee.png">

 - [OPTIONAL] The bottom whitespace is due to the `100vh` value of the `.container` class, and the side whitespace is due to the fixed width value. Try changing them to better understand the behavior in your project.